graveyard.json

```
@@ -1,4 +1,12 @@
 [
+  {
+    "dateClose": "2022-12-24",
+    "dateOpen": "2020-10-16",
+    "description": "Google Taskmate was a crowdsourcing platform developed by Google that allows users to earn money by completing tasks on their smartphones. It was launched in 2020.",
```

@codyogden
Here you go, I fixed it for you.

```suggestion
    "description": "Google Taskmate was a crowdsourcing platform developed by Google that allows users to earn money by completing tasks on their smartphones.",
```